website/docs/integrations/services/bookstack/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Bookstack
+---
+
+## What is Bookstack
+
+From https://en.wikipedia.org/wiki/BookStack
+
+:::note
+BookStack is a free and open-source wiki software aimed for a simple, self-hosted, and easy-to-use platform. Based on Laravel, a PHP framework, BookStack is released under the MIT License. It uses the ideas of books to organise pages and store information. BookStack is multilingual and available in over thirty languages. For the simplicity, BookStack is considered as suitable for smaller businesses or freelancers.
+:::
+
+:::note
+This is based on authentik 2021.7.2 and BookStack V21.05.3. Instructions may differ between versions.
+:::
+
+## Preparation
+
+The following placeholders will be used:
+
+- `book.company` is the FQDN of BookStack.
+- `authentik.company` is the FQDN of authentik.
+- `METADATAURL` is the url for the SAML metadata from authentik
+
+### Step 1
+
+In authentik, under _Providers_, create a _SAML Provider_ with these settings:
+
+**Protocol Settings**
+- Name: Bookstack
+- ACS URL: https://book.company/saml2/acs
+- Issuer: https://authentik.company
+- Service Provider Binding: Post
+- Audience: https://book.company/saml2/metadata
+**Advanced protocol settings**
```

Comment:
I think you need some additional line breaks here, Markdown puts the **advanced protocol settings** on the audience line

Reply:
Totally missed that when I looked at the preview.  It's fixed Thanks